e2e/utils.ts

```diff
@@ -188,6 +191,10 @@ export const makeTempDir = (prefix: string): string => {
 };
 
 export const prepareStandaloneSetup = (fixtureName: string) => {
+  if (process.env.TEST_FORCE_NORMAL_SETUP) {
+    return prepareNormalSetup(fixtureName) as any;
+  }
```

Comment:
As I said, if we can use normal setup instead of standalone setup for some specs, let's do it.

Reply:
Reverted